packages/client/src/containers/CollectionsPage/CollectionsPage.Container.js

```diff
@@ -5,7 +5,7 @@ import DropDownViewList from '../../components/ProductListDropDown/DropDownViewL
 export const CollectionsPage = () => {
   return (
     <div>
-      <DropDownViewList extra />
+      <DropDownViewList filterIcon />
```

Comment:
This is a bit better but maybe this should be something like "showFilterIcon" since it's a boolean after all. 
But I also wonder why it's hard coded here to true?

Also ask yourself if it really makes sense to name this prop the same in all the layers it's being passed through, does it mean the same basic thing in all those components? It's okay to rename them so that they make the most sense in the context of each individual component.

Reply:
Thanks for the help with the name for prop.
In my opinion, the same name, on the contrary, makes it easier to understand what kind of prop it is. It is always true because it is always used for one select.
Perhaps I did not understand something from your message, so I would be grateful for an explanation